src/coffee/bootstrap-tour.coffee

```diff
@@ -475,7 +477,8 @@
 
       $element
       .popover(
-        placement: step.placement
+        placement: ->
+          return if step.smartPlacement is true then "auto #{step.placement}" else step.placement
```

Comment:
I meant, the function, not the code inside it. Why not put the following instead

```placement: if step.smartPlacement is true then "auto #{step.placement}" else step.placement```